### PR TITLE
adding a few examples to Bio/Seq.py

### PR DESCRIPTION
**Pull request by azneto**
This pull request addresses issue #1203

I hereby agree to dual licence this and any previous contributions under both
the _Biopython License Agreement_ AND the _BSD 3-Clause License_.

I am happy to be thanked by name in the ``NEWS.rst`` and
``CONTRIB.rst`` files.

I have read the ``CONTRIBUTING.rst`` file and understand that AppVeyor and
TravisCI will be used to confirm the Biopython unit tests and ``flake8`` style
checks pass with these changes.

### CODE REVIEW

File: Bio/Seq.py [hunk 1]
```diff
@@ -228,7 +228,12 @@ def __len__(self):
         return len(self._data)  # Seq API requirement
 
     def __getitem__(self, index):  # Seq API requirement
-        """Return a subsequence of single letter, use my_seq[index]."""
+        """ Return a subsequence of single letter, use my_seq[index].
```
Comment (peterjc) [inline Bio/Seq.py]: Unwanted space before "Return"

Comment (peterjc) [reply Bio/Seq.py]: This is why the TravisCI style checks are failing,

```
$ flake8 Bio/
Bio/Seq.py:230:1: D210 No whitespaces allowed surrounding docstring text
```

See https://github.com/biopython/biopython/blob/master/CONTRIBUTING.rst for how to setup ``flake8`` with the relevant plugins.

**Comment by peterjc** [comment]
How do you want to be named in the ``NEWS.rst`` and ``CONTRIB.rst`` files? You can add yourself as part of this pull request if you like - not the alphabetic sorting though.

**Comment by azneto** [comment]
This is such a minor thing. I'd rather continue to submit pull requests to get the grip and we can add my name further on. 
BTW, if I do new commits to this branch will it be automatically included in this pull request or do I need to create a new one?

**Comment by peterjc** [comment]
Unless less the pull request has been merged, you can add more commits to the branch, push the branch to github, and it will update the pull request.

I'm happy with the changes proposed, so will merge this as is. Feel free to make a new branch and new pull request for future contributions, and we can add your name when you feel comfortable.

**Comment by peterjc** [comment]
Merged, thank you 👍

**Comment by azneto** [comment]
:+1:

**Pull request merged**